ledger-tool/src/main.rs

```diff
@@ -2370,7 +2372,16 @@ fn main() {
                     }
 
                     if child_bank_required {
-                        bank.fill_bank_with_ticks_for_tests();
+                        let num_ticks_per_slot = bank.ticks_per_slot();
+                        let num_hashes_per_tick = bank.hashes_per_tick().unwrap_or(0);
+                        let parent_blockhash = bank.last_blockhash();
+                        let tick_entries =
+                            create_ticks(num_ticks_per_slot, num_hashes_per_tick, parent_blockhash);
+
+                        let scheduler = BankWithScheduler::no_scheduler_available();
+                        tick_entries.iter().for_each(|tick_entry| {
+                            bank.register_tick(&tick_entry.hash, &scheduler);
+                        });
```

Comment:
This is definitely more corrrect than before.

Want to just point out that this method of adding ticks will not propagate them to the blockstore as they normally would be. That's fine for creating a snapshot afaict.
I don't think there's another use case for this needing to be properly done either, so the risk of someone copying this when ticks DO need to be in blockstore is probably low.